tests/functional/spec/features/hub_and_spoke/hub_and_spoke_required_enable.spec.js

```diff
@@ -0,0 +1,18 @@
+import HouseholdRelationshipsBlockPage from "../../../generated_pages/hub_section_required_and_enabled/household-relationships-block.page";
+import RelationshipsCountPage from "../../../generated_pages/hub_section_required_and_enabled/relationships-count.page";
+
+describe("Hub and spoke section required and enabled", () => {
+  beforeEach("Load the survey", () => {
+    browser.openQuestionnaire("test_hub_section_required_and_enabled.json");
+  });
+  it(" When I answer 'Yes' to the first section, meaning the second section is enabled,Then I should be redirected to the relationship count section", () => {
```

Comment:
Probably need to mention that the 2nd section is relationship count, could even get away with not saying that. "I am routed to the second section" The key is you are going to the enabled one, at the moment I can't quite tell that from "redirected to the relationship count section" what it is doing